src/app/devices/kvm/kvm.component.ts

```diff
@@ -54,13 +56,14 @@ export class KvmComponent implements OnInit, AfterViewInit, OnDestroy {
       this.isLoading = true
       this.deviceId = params.id
     })
-    this.devicesService.stopwebSocket.subscribe(() => {
+    this.stopSocketSubscription = this.devicesService.stopwebSocket.subscribe(() => {
       this.stopKvm()
     })
 
-    this.devicesService.startwebSocket.subscribe(() => {
+    this.startSocketSubscription = this.devicesService.connectKvmSocket.subscribe(() => {
```

Comment:
suggest to rename `connectKvmSocket` to `connectKVMSocket`

Reply:
Renamed the variable to 'connectKVMSocket'